### PR TITLE
add dockerfles for R runtimes

### PR DESCRIPTION
**Pull request by ablack3**

### CODE REVIEW

File: execution-environments/darwin-base/azure-pipeline.yml
Comment (konstjar) [inline execution-environments/darwin-base/azure-pipeline.yml:1]: Do we need this file in Arachne repo?

Comment (dmitrys-odysseus) [reply execution-environments/darwin-base/azure-pipeline.yml:1]: This is the definitely a wrong place to put these files. 
Images are are not coupled with arachne datanode and can be used without it.
If we need them under version control, it should be either https://github.com/OHDSI/ArachneExecutionEngine or separate repository.

**Review by dmitrys-odysseus — CHANGES_REQUESTED** [review]
This is the definitely a wrong place to put these files.
Images are are not coupled with arachne datanode and can be used without it.
If we need them under version control, it should be either https://github.com/OHDSI/ArachneExecutionEngine or separate repository.

File: execution-environments/darwin-base/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,96 @@
+FROM --platform=linux/amd64 rocker/rstudio:4.2
```
Comment (dmitrys-odysseus) [inline execution-environments/darwin-base/Dockerfile:1]: Images should be kept small and I don't see why whole rstudio should be there. 
Can we change this to some image that only contains R runtime?

Comment (ablack3) [reply execution-environments/darwin-base/Dockerfile:1]: yes

File: execution-environments/darwin-base/azure-pipeline.yml
Comment (ablack3) [reply execution-environments/darwin-base/azure-pipeline.yml:1]: nope

File: execution-environments/darwin-base/Dockerfile (hunk 1 above)
Comment (ablack3) [reply execution-environments/darwin-base/Dockerfile:1]: The use case for including RStudio is to provide debugging tools but I agree in the ideal case (hopefully 90% of the time) there is no need for RStudio.

**Comment by ablack3** [comment]
closing this... move to Arachne-execution-engine repo instead.

**Pull request closed without merging**